informus/rules/Delirium.md

```diff
@@ -111,11 +111,11 @@ b) NO Valid RASS documented with a CAM-ICU score documented means that at the ti
   
 *the following feeds into the (i) front tile - 24-hour rolling window and (ii) SPC patients with delirium chart*
 
-**[C]  overall CAM-ICU front tile calculation: Patients with Delirium in the last 24 hours**
-- This is a live update that looks at incidence of delirium in the last 24 hours (inpatients and those discharged)
+**[C]  overall CAM-ICU front tile calculation: Patients with Delirium in the last 2 (completed) shifts**
+- This is a live update that looks at incidence of delirium in the last 2 complted shifts (inpatients and those discharged) - RATIONALE - only one assessment is required per shift, therefore an incomplete will not reflect the quantity of delirium on a ward accurately
 - Refer to CAM-ICU validity rules above (use backfilled CAM-ICU) *even if the last CAM-ICU was documented over 24 hours ago, it was still valid until the end of that shift so should be included*
-- Numerator: the number of patients present on on each unit at any time over the last 24 hours  with positive CAM-ICU scores in the last 24 hours 
-- Denominator: the number of patients present on on each unit at any time over the last 24 hours 
+- Numerator: the number of patients present on on each unit at any time over the last 2 complted shifts with positive CAM-ICU scores
+- Denominator: the number of patients present on on each unit at any time over the last 2 complted shifts 
```

Comment:
If I understand correctly, neither the numerator nor the denominator care about filtering by the RASS criteria (i.e. between -3 to + 4)? 

So the numerator does not care about the RASS scores, and the denominator will just be all patients who were on the unit over the last 2 shifts, again regardless of their RASS scores?